homeassistant/components/wirelesstag/__init__.py

```diff
@@ -131,8 +132,7 @@ def install_push_notifications(self, binary_sensors):
     def local_base_url(self):
         """Define base url of hass in local network."""
         if self._local_base_url is None:
-            self._local_base_url = f"http://{util.get_local_ip()}"
-
+            self._local_base_url = f"http://{self.hass.loop.run_until_complete(async_get_source_ip(self.hass, target_ip=PUBLIC_TARGET_IP))}"
```

Comment:
Let's defer this one until we can refactor `def setup` to be `async def async_setup` and avoid all the context switching

Reply:
Done